flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/rules/logical/ConvertToNotInOrInRule.scala

```diff
@@ -184,6 +188,54 @@ class ConvertToNotInOrInRule
       case _ => rexNodes.size >= THRESHOLD
     }
   }
+
```

Comment:
Could we use the following method to fix?
![image](https://user-images.githubusercontent.com/1525333/141961739-0592f735-f466-42f3-a571-a0de2b0c56df.png)

Reply:
Sorry, I checked the scalar code in currently `ConvertToNotInOrInRule.scala`, they are same with the above java code.
Could we move this method to `FlinkRexBuilder`, and overwrite the makeIn method of parent class?
Then we could avoid this bug appears in other places.